### PR TITLE
Modify/Fix ConsoleLogHandler to output desired console output

### LINKED ISSUE #2321: ConsoleLogHandler does not abide by consoleLogLevel

### PR DESCRIPTION
**Pull request by Channyboy**
Modify/Fix ConsoleLogHandler  to print out log messages based on consoleLogLevel and to print out SystemOut and SystemErr only if copysystemstreams is set
#build 
fixes #2321

### CODE REVIEW

**Comment by Channyboy** [comment]
#build

File: dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/ConsoleLogHandler.java [hunk 1]
```diff
@@ -79,6 +81,13 @@ public void synchronousWrite(Object event) {
                 (loggerName.equalsIgnoreCase(LoggingConstants.SYSTEM_OUT) || loggerName.equalsIgnoreCase(LoggingConstants.SYSTEM_ERR))) {
                 sysLogHolder.getOriginalStream().println(messageOutput);
             }
+
+            //Other wise we will be writing out  accessLog or ffdc or trace
```
Comment (donbourne) [inline dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/ConsoleLogHandler.java]: seems like this comment is backwards... this addition is to allow accessLog / ffdc / trace to appear in console output, correct?

**Comment by donbourne** [comment]
latest change (to add trcfilename == stdout) not needed per design.  tracefilename=stdout does not apply when consoleLogFormat=json

**Comment by Channyboy** [comment]
#build reverted changes and building again

**Comment by fmhwong** [comment]
#build

**Pull request closed without merging**